google/ggrc-core
# GGRC-2474 Remove person global cad

# Dependencies
we should fix selenium tests. 
- [ ] https://github.com/google/ggrc-core/pull/9207
# Issue description

We have to remove person global custom attribute.
Because we have none in prod, we decided that we will not migrate existing ones. we'll just prevent the creation of new ones and then manually delete such attributes on test instances.

# Steps to test the changes

Check that you can't create new person global cad.
check that local cads work fine for assessments and assessment templates
check that you can delete existing global person cads.

# Solution description

this is just post flush hook for cad type check. There are also some changes so that detection of global cads works on assessment templates.

# Sanity checklist




# PR Review checklist
